### ISSUE TITLE
Expected a string object but getting a list object instead

### ISSUE THREAD

**Issue by rounak-meyur** [openEDI/oedisi-example#84]
We have a generator in a user-defined DSS file as follows
```typescript
new generator.HRSST_PV_PMU  Phases=3   Bus1=HRSST_PV   kV = 0.48 conn=wye Model=1  kW=3049.66   kVAr=-109.69
```

Line 541 in the python script `LocalFeeder/FeederSimulator.py` 
```python
bus = gen["bus"].split(".")
```
generates an error since `gen["bus"]` is a `list` instead of a `string`. 

A debug statement shows that the value of `gen["bus"]` is `['hrsst_pv', '1', '2', '3']`

We used a work around as follows
```python
if not isinstance(gen["bus"], list):
    bus = gen["bus"].split(".")
else:
    bus = gen["bus"]
```

Are we defining the generator in the DSS file in an incorrect way?

**Comment by Tylores** [comment]
Looking into this it looks like EPRI has some models as well that don't follow the convention of Phases=3 bus=name.1.2.3 and just use bus=name. 

```
New PVSystem.3P_ExistingSite4 phases=3 bus1=B51854_sec2     kV=0.416 kVA=523 irradiance=1 Pmpp=475 pf=1 %cutin=0.1 %cutout=0.1 	!yearly=PV_16_15
```

**Comment by josephmckinsey** [comment]
Fixed in #86
